jetty-ee10/jetty-ee10-osgi/pom.xml

```diff
@@ -117,11 +106,36 @@
         <artifactId>org.eclipse.osgi</artifactId>
         <version>${osgi-version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.osgi</groupId>
+        <artifactId>org.osgi.service.cm</artifactId>
+        <version>${osgi-service-cm-version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.osgi</groupId>
+        <artifactId>org.osgi.service.component</artifactId>
```

Comment:
this and some other repeated would be better in dptMngt in top pom and added here without version (to have a single place with the version and dptMngt is a better way to handle transitivity and possible upper bound issues)

Reply:
Lets tackle this as a clean PR in a followup PR